src/components/Comments.jsx

```diff
@@ -0,0 +1,63 @@
+import { getComments } from "../../api";
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import moment from "moment";
+
+const Comments = () => {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    setError(null);
+    getComments(article_id)
+      .then((data) => {
+        setComments(data.comments);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        setError("error fetching data");
+        setIsLoading(false);
+      });
+  }, [article_id]);
```

Comment:
If you do the above comment then there is no need for this to be a dependancy as the useEffect will run everytime the article page component re-renders